### DISCUSSION TITLE
Required to restart dev server anytime I create a new route file.

### DISCUSSION THREAD

**Discussion by KwaminaWhyte** [remix-run/remix#6362] [General]
Started a new project with v1.16.0, using the v2 route convention.
I am forced to stop the server and restart it whenever I create a new route file or rename a route file for it to take effect.

Any help will be appreciated.

**Comment by arielvieira** [comment]
Same issue here.

**Comment by w0nche0l** [comment]
remix newbie here, running into the same issue.
